ratings/helm/templates/database-secrets.yaml

```diff
@@ -1,14 +1,14 @@
-{{ if .Values.db }}
+{{- if .Values.db.deploy -}}
 apiVersion: v1
 kind: Secret
 metadata:
-  name: gpte-db-secrets
-  namespace: {{ include "babylon-ratings.namespaceName" . }}
+  name: {{ .Values.db.secretName }}
+  namespace: {{ include "reportingCatalogItem.namespaceName" . }}
```

Comment:
i think should be the babylon-ratings namespace

Reply:
Yes, I will fix!